library/src/scripts/messages/messageStyles.ts

```diff
@@ -95,6 +95,24 @@ export const messagesClasses = useThemeCache(() => {
         zIndex: 20,
     });
 
+    const innerWrapper = style("innerWrapper", {
+        $nest: {
+            "&&": {
+                flexDirection: "unset",
+                maxWidth: "max-content",
```

Comment:
This isn't an actual finalized CSS feature. It seems to only be part of a draft specification.

https://drafts.csswg.org/css-sizing-3/#width-height-keywords

Reply:
deleted this styling, thanks for clarification